components/support/rc_crypto/src/lib.rs

```diff
@@ -55,7 +55,6 @@ pub use crate::error::{Error, ErrorKind, Result};
 /// Only required to be called if you intend to use this library in conjunction
 /// with the `hawk` or the `ece` crate.
 pub fn ensure_initialized() {
-    nss::ensure_initialized();
```

Comment:
Should this become `expect_nss_initialized()?`

Reply:
Good catch!

If we'd change the signature `pub fn ensure_initialized() -> Result<()> {` this result would propagate all over the place, for example to [HawkRequestBuilder::new](https://github.com/mozilla/application-services/blob/main/components/fxa-client/src/internal/http_client.rs#L608-L616). I think that's fine, I'd go ahead and make those changes if you agree

Reply:
done